cwms-data-api/src/main/java/cwms/cda/api/CatalogController.java

```diff
@@ -129,6 +136,15 @@ public void getAll(Context ctx) {
                 @OpenApiParam(name = BOUNDING_OFFICE_LIKE, description = "Posix <a href=\"regexp.html\">regular expression</a> "
                         + "matching against the location bounding office. "
                         + "When this field is used items with no bounding office set will not be present in results."),
+                @OpenApiParam(name = Controllers.INCLUDE_EXTENTS, type = Boolean.class,
+                        description = "Whether the returned catalog entries should include timeseries "
+                                + "extents. Only valid for TIMESERIES. Default is true."),
+                @OpenApiParam(name = Controllers.EXCLUDE_EMPTY, type = Boolean.class,
```

Comment:
Would it make sense to invert this condition? E.g. only include those that have data?
We do have to join a table so maybe a performance hit, but I know that's cause some confusion in the past.

@krowvin @DanielTOsborne @adamscarberry

Reply:
I need a decision on this.   It would make sense to me for the flag to be include-empty or exclude-empty.  It sounds like more than one person expected the results to exclude empty items by default so maybe that is the default behavior.

Reply:
Exclude empty by default, It'll cause less confusion overall with the way publicly available versions are setup.